digicampipe/calib/camera/r1.py

```diff
@@ -3,7 +3,7 @@
 
 
 def calibrate_to_r1(event_stream, dark_baseline):
-
+    dark_baseline = dark_baseline['baseline']
```

Comment:
Thanks for the improvements

Reply:
I would rather prefer actually that we pass the `dark_baseline['baseline']` instead of the `dict()` /`.npz` file containing the baseline (`dark_baseline`). I think this solves the issue (i.e. loaded once outside the function) correct?

Also I think it is more understandable to pass an array rather than a dict.
What do you think?

Reply:
Yes, I think what you are saying makes sense. Just for a performance improvement I wanted to make a minimal change .. since making changes in many places just to improve performance is in my opinion very bad style.

What you propose is to make a change, so that the code is better understandable in the end, and I think that warrants for a change in many places ... 

(To be honest I do not even know in how many places we would have to change something.)

Reply:
So ... is it okay when we do this change in another PR?